src/main/java/eu/decentsoftware/holograms/api/utils/items/SkullUtils.java

```diff
@@ -118,17 +120,32 @@ public static String getTexture(@NonNull ItemStack itemStack) {
 	 * Set the Base64 texture of the given skull ItemStack.
 	 *
 	 * @param itemStack The ItemStack.
-	 * @param texture   The new skull texture (Base64).
+	 * @param texture   The new skull texture (Base64 or URL).
```

Comment:
It's not really "or URL". It can either be a very specific URL: `http://textures.minecraft.net/texture/` or a hexidecimal *id?* of a texture on the same site. Just mentioning URL can be misleading here.